traitsui/testing/tester/locator.py

```diff
@@ -0,0 +1,26 @@
+""" This module defines objects for locating nested UI targets, to be
```

Comment:
Just noticed this file is missing the copyright header

Reply:
Thanks!